src/components/TSFeatureJobCard/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Card, Row, Col } from 'antd';
+import classnames from 'classnames';
+import './index.less';
+import TSButton from '../../components/TSButton';
+import IconMaker from '../../assets/images/icon-map-maker.svg';
+import IconLove from '../../assets/images/icon-love.svg';
+
+class TSFeatureJobCard extends PureComponent {
+  render() {
+    const { className, ...props } = this.props;
+
+    return (
+      <Card hoverable {...props} className={classnames('ts-feature-job-card', 'ts-feature-job-card--border-top', className)}>
+        <Row>
```

Comment:
Is it component done already? Or just for demo purpose? Cause I see you're set fixed data for component

Reply:
All of the components just hardcode data to test UI, I will change it to dynamic.

Reply:
okay, I see! then maybe let wait after you updated to dynamic, then we can merge this PR